README.md

```diff
@@ -76,6 +62,34 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Use Base Profiles to access API Mediation Layer
+
+Leverage existing base profiles with a token to log in to API Mediation Layer.
+
+Before using the base profile functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) v1.13.0 or higher installed.
+
+**Follow these steps:**
+
+1. Open Zowe CLI and run the following command:
+
+   ```
+   zowe auth login apiml
+   ```
+
+2. Follow the instructions to complete the login and provide all necessary information to create a base profile with a token.
```

Comment:
I would just delete "and provide all necessary info..." because it sounds like two different things. In reality, the login process has you do this, so just the beginning is needed.